### ISSUE TITLE
how to monitor/print active (decaying) learning rate of RectifiedAdam + Lookahead (Ranger)

### ISSUE THREAD

**Issue by turmeric-blend** [tensorflow/addons#1946]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04
- TensorFlow version and how it was installed (source or binary): TF 2.1 installed via Conda
- TensorFlow-Addons version and how it was installed (source or binary): TF-addons 0.9 installed via pip
- Python version: Python 3.7
- Is GPU used? (yes/no): Yes

**Describe the bug**

No clear way to monitor/print the active (decaying) learning rate of the following setup:

```
optimizer = tfa.optimizers.RectifiedAdam(lr=0.0001)
optimizer = tfa.optimizers.Lookahead(optimizer, sync_period=6, slow_step_size=0.5)
```

`optimizer.lr` currently returns the initial learning rate, 0.0001.

**Comment by bhack** [comment]
You need to use something like https://www.tensorflow.org/guide/keras/custom_callback#learning_rate_scheduling

**Comment by kelvinkoh0308** [comment]
You can get by using the following code

`optimizer._optimizer._decayed_lr(tf.float32)`

**Comment by seanpmorgan** [comment]
Closing. Please let us know if there are still issues.
